Diary/src/main/java/com/microsoft/device/display/samples/diary/ui/pages/CalendarPage.kt

```diff
@@ -0,0 +1,124 @@
+package com.microsoft.device.display.samples.diary.ui.pages
+
+import android.view.ViewGroup
+import android.widget.CalendarView
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Edit
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shadow
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.viewinterop.AndroidView
+import com.microsoft.device.display.samples.diary.R
+import com.microsoft.device.dualscreen.twopanelayout.TwoPaneScope
+import java.time.LocalDate
+
+/**
+ *  The CalendarView Composable which also contains the Diary Content for the selected day
+ */
+@Composable
+fun TwoPaneScope.CalendarPage(
+    content: String,
+    selectedDate: LocalDate,
+    updateDate: (LocalDate) -> Unit,
+    updateContent: () -> Unit
+) {
+    val twoPaneScope = this
+    Scaffold(
+        topBar = {
+            TopAppBar(
+                title = {
+                    Text(text = stringResource(id = R.string.app_name))
+                },
+                backgroundColor = MaterialTheme.colors.primaryVariant,
+                actions = {
+                    if (twoPaneScope.isSinglePane) {
+                        IconButton(
+                            onClick = { twoPaneScope.navigateToPane2() }
```

Comment:
you may want to change how you navigate between panes, as the new twopanelayout navigation does not use navigateToPane2 anymore and once merged, might have conflicts

Reply:
actually for the new original  TwoPaneLayout (not TwoPaneLayoutNav), this will be ok! we made sure to update to use `TwoPaneScope` so everything should be recognized 😄